suite-native/storage/src/storage.ts

```diff
@@ -39,33 +53,37 @@ export const clearStorage = () => {
     RNRestart.restart();
 };
 
-// Ideally it should never happen but we need to be sure that at least some message is displayed.
-// If someone will mess with encryptionKey it can corrupt storage and app will crash on startup.
-// Then app will hang on splashscreen indefinitely so we at least want to show some error message.
+const alertUser = () => {
+    SplashScreen.hideAsync();
```

Comment:
🐰 is partially correct here, it's not clear why there duplication of hiding splashscreen, please add comment why

Reply:
comment added, does it makes better sense? https://github.com/trezor/trezor-suite/pull/16819/commits/ea99dc780716c8dcfc56aea1db8b0664f64646e0